### PR TITLE
Rotation implementation

### PR DESCRIPTION
**Pull request by rafael-chavez**

### CODE REVIEW

**Comment by facebook-github-bot-5** [comment]
@rafael-chavez updated the pull request.

**Comment by rafael-chavez** [comment]
@marekcirkos

**Comment by facebook-github-bot-5** [comment]
@rafael-chavez updated the pull request.

**Comment by facebook-github-bot-5** [comment]
@rafael-chavez updated the pull request.

**Comment by rafael-chavez** [comment]
@marekcirkos

File: WebDriverAgentLib/Commands/FBOrientationCommands.m [hunk 1]
```diff
@@ -84,16 +101,31 @@ + (BOOL)setDeviceOrientation:(NSString *)orientation forApplication:(FBApplicati
...
+}
 
-  // We have a busy loop here while we wait for the orientation to change as we do not have any hooks
-  // into the event being handled.
-  // If we could just hook into the event handler to know when it has been processed..
-  NSDate *startDate = [NSDate date];
-  while (![[self interfaceOrientationForApplication:application] isEqualToString:orientation] && (-1 * [startDate timeIntervalSinceNow]) < kFBWebDriverOrientationChangeDelay) {
-    CFRunLoopRunInMode(kCFRunLoopDefaultMode, 0.0, YES);
-  }
++ (BOOL)setDeviceRotation:(NSDictionary *)rotationObj forApplication:(FBApplication *)application
```
Comment (marekcirkos) [inline WebDriverAgentLib/Commands/FBOrientationCommands.m]: Looks good to me. However could you move those methods to XCUIDevice category and write an integration test for it?

**Comment by facebook-github-bot-9** [comment]
@rafael-chavez updated the pull request.

**Comment by facebook-github-bot-9** [comment]
@rafael-chavez updated the pull request.

**Comment by facebook-github-bot-9** [comment]
@rafael-chavez updated the pull request.

**Comment by rafael-chavez** [comment]
@marekcirkos updated!

**Comment by facebook-github-bot-1** [comment]
@rafael-chavez updated the pull request.

**Comment by facebook-github-bot-9** [comment]
@rafael-chavez updated the pull request.

**Comment by marekcirkos** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-5** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/311644589223511/int_phab) to review internal test results.

**Pull request closed without merging**